test.js

```diff
@@ -72,6 +75,36 @@ test('should start the server at the given prefix', t => {
   })
 })
 
+test('should start fastify at given socket path', t => {
+  t.plan(2)
+
+  const sockFile = path.join(__dirname, '/test.sock')
```

Comment:
Probably `path.resolve` is better :)
Checkout how we used it [here](https://github.com/fastify/point-of-view/blob/master/index.js) :)

Reply:
done